test/unit/validator.test.js

```diff
@@ -78,26 +78,26 @@ describe('validator', () => {
     });
   });
 
-  afterEach(async () => {
```

Comment:
`async () => {}` with a space after `async` should be preferred. Was there a rule change in eslintrc?

Reply:
I added an eslint rule `space-before-function-paren` to avoid `funcName   (){}` , I'll remove it